.github/workflows/go.yml

```diff
@@ -29,23 +29,46 @@ jobs:
   e2e-test:
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
       matrix:
-        test-suite: [TestEthereumE2ESuite, TestFabricE2ESuite]
         blockchain-provider: [geth, fabric]
-        token-provider: [none, erc1155, erc20_erc721]
-        database-type: [sqlite3, postgres]
-        exclude:
-          - blockchain-provider: geth
-            test-suite: TestFabricE2ESuite
-          - blockchain-provider: fabric
-            test-suite: TestEthereumE2ESuite
-          - blockchain-provider: fabric
-            token-provider: erc1155
-          - blockchain-provider: fabric
-            token-provider: erc20_erc721
-          - blockchain-provider: geth
-            token-provider: none
-      fail-fast: false
+        exclude: [ blockchain-provider: geth, blockchain-provider: fabric ]
```

Comment:
Confused about why 2 blockchain providers are defined and then both excluded... is this necessary for the "whitelist" below to work?

Reply:
Yeah this was a bit strange. Basically I wanted a list of all the permutations spelled out exactly (fewer lines of code). But you can't just give `include` by itself without first having a matrix to start from. This was basically a hack to create an empty matrix.

Reply:
OK, will leave it since it seems to work. Perhaps we'll find a cleaner way some other day 😁